utils/fast-math/logs.hpp

```diff
@@ -27,9 +27,9 @@
 // for negligible cost in accuracy
 #ifdef PORTABILITY_STRATEGY_KOKKOS
 #define BD_USE_FMATH 0
-#else
-#include <fmath.hpp>
+#elif SINGULARITY_USE_FMATH
 #define BD_USE_FMATH 1
+#include <fmath.hpp>
```

Comment:
Should there be an `#else` that defines `BD_USE_FMATH 0`?

Reply:
See d75a0b6

Reply:
:+1:

The `#define` and then `#if (...) #undef` paradigm feels a little backwards to me but it works and it's clear so makes sense to me.